app/src/src/navigators/AuthNavigator.jsx

```diff
@@ -84,16 +85,19 @@ const AuthNavigator = () => {
     // Render unauthenticated screens
     return (
       <NavigationContainer>
-        <Stack.Navigator>
+        <Stack.Navigator
+          screenOptions={{ headerShown: false }}
+        >
```

Comment:
I think this is causing formatting errors, you can change to be on one line and then I think the checks will pass

Reply:
also because header is gone, the images on the login page (only ios) and password reset page (both) are off the screen. can you adjust the styles on the login page and password reset page so that the images are on the screen (like the registration page)?

Reply:
Hi, thanks for the check, I just fixed the formatting! However, all the images are lined up and visible on android (I have a Samsung S7). I'm not sure how to adjust the styles without IOS... perhaps there's another way to make it responsive on all screens?

Reply:
hmm yeah instead of adjusting the styles, maybe you try to add `contentContainerStyle={{ flex: 1 }}` to both of the KeyboardAwareScrollView on the login page and password reset page: `<KeyboardAwareScrollView contentContainerStyle={{ flex: 1 }}>`

Reply:
Got it, thanks so much!!